draft-irtf-cfrg-frost.md

```diff
@@ -464,19 +445,19 @@ structures into values that can be processed with hash functions.
 
 ~~~
   Inputs:
-  - commitment_list = [(j, D_j, E_j), ...], a list of commitments issued by each signer,
-    where each element in the list indicates the signer index and their
-    two commitment Element values. B MUST be sorted in ascending order
+  - commitment_list = [(i, x_i, y_i), ...], a list of commitments issued by each signer,
+    where each element in the list indicates the signer index i and their
+    two commitment Element values (x_i, y_i). This list MUST be sorted in ascending order
```

Comment:
I worry a little about the use of `x` and `y` because that may be confused with `x`-coordinates / `y`-coordinates of curve group elements (points) in affine coordinates

Reply:
Yeah, I didn't want to use D/E though, since we dropped that notation. Happy to take suggestions!